index.html

```diff
@@ -5,55 +5,55 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Mirco Warnck</title>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="./styles/main.css" />
   </head>
 
   <body>
-    <header>
+    <header class="header">
       <h1>Mirco Warnck</h1>
-      <nav>
-        <ul>
+      <nav class="nav">
+        <ul class="nav__ul">
           <li>
-            <a class="nav-links" href="#biography">Biography</a>
+            <a class="nav__nav-links" href="#biography">Biography</a>
           </li>
           <li>
-            <a class="nav-links" href="#contact">Contact</a>
+            <a class="nav__nav-links" href="#contact">Contact</a>
           </li>
           <li>
-            <a class="nav-links" href="#networks">Networks</a>
+            <a class="nav__nav-links" href="#networks">Networks</a>
           </li>
         </ul>
       </nav>
     </header>
     <main>
-      <section id="image-skills" class="wrapper">
-        <figure class="illustration">
+      <section class="image-skills wrapper" id="image-skills">
+        <figure class="image-skills__illustration">
           <img
-            class="header-img"
+            class="image-skills__header-img"
```

Comment:
hmmmh i would think it´s a header or an h tag and not a photo... maybe better for eg: `image-skills__photo` !? 
and for the figure tag it might be easier if you name it for eg `image-skills_figure` 
what u think?

Reply:
sorry i didn´s see the img tag at the end :monkey_face:

Reply:
sorry i didn´s see the img tag at the end 🐵 and i mean  `image-skills__figure` with 2 underscores

Reply:
Hey Michi, what a great feedback! I make it easy and sticked to the former class names but your ideas would make it better and clear for other viewers. Thank you very much! :)